python/ccf/clients.py

```diff
@@ -452,7 +474,14 @@ def request(
         global_commit = unpack_seqno_or_view(out[18:26])
         payload = out[26:]
         if status_code == 200:
-            body = json.loads(payload) if payload else None
+            if payload:
+                # Ideally, the content type should be sent like for HTTP.
```

Comment:
The view here is that websockets is the efficient option, and it is the efficient option because it does away with the large variable header. Endpoints are expected to expose their schema out of band rather than send (or expect) self-describing traffic.

An application that wishes to replicate the content-type concept across their websockets endpoints would do so in the outer frame of their schema.

Reply:
So, is it fine to keep the try-except? If not, then maybe for WebSocket it should always return `bytes`?

Reply:
We probably should, but then we have to adjust the logging test accordingly. Let's keep it as is and do that clean up in a separate PR?

Reply:
https://github.com/microsoft/CCF/issues/1584